src/main/java/dk/cngroup/lentils/controller/FinalPlaceController.java

```diff
@@ -22,31 +24,35 @@
     private static final String REDIRECT_FINALPLACE_FORM = "redirect:/admin/finalplace/";
 
     private FinalPlaceService finalPlaceService;
+    private ModelMapperWrapper mapper;
 
     @Autowired
-    public FinalPlaceController(final FinalPlaceService finalPlaceService) {
+    public FinalPlaceController(final FinalPlaceService finalPlaceService,
+                                final ModelMapperWrapper modelMapperWrapper) {
         this.finalPlaceService = finalPlaceService;
+        this.mapper = modelMapperWrapper;
     }
 
     @GetMapping(value = "/")
     public String finalPlace(final Model model) {
-        model.addAttribute("finalPlace", finalPlaceService.getFinalPlace());
+        model.addAttribute("finalPlace", mapper.map(finalPlaceService.getFinalPlace(), FinalPlaceFormDTO.class));
         model.addAttribute(
                 "serverTime",
                 LocalDateTime.now().format(DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm")));
         return VIEW_FINALPLACE_FORM;
     }
 
     @PostMapping(value = "/update")
-    public String saveFinalPlace(@Valid @ModelAttribute final FinalPlace finalPlace,
+    public String saveFinalPlace(@Valid @ModelAttribute("finalPlace") final FinalPlaceFormDTO finalPlaceFormDto,
                                  final BindingResult bindingResult,
                                  final Model model) {
-        finalPlaceService.checkFinishTimeBeforeResultsTime(bindingResult, finalPlace);
+        finalPlaceService.checkFinishTimeBeforeResultsTime(bindingResult,
+                mapper.map(finalPlaceFormDto, FinalPlace.class));
         if (bindingResult.hasErrors()) {
-            model.addAttribute("finalPlace", finalPlace);
+            model.addAttribute("finalPlace", finalPlaceFormDto);
             return VIEW_FINALPLACE_FORM;
         }
-        finalPlaceService.save(finalPlace);
+        finalPlaceService.save(mapper.map(finalPlaceFormDto, FinalPlace.class));
```

Comment:
Toto by jsi mohl zavolat jen jednou a ulozit si to do premenne a pouzit to i o par radku vyse.